.travis.yml

```diff
@@ -50,6 +50,7 @@ env:
     - 'TESTDIR="Java/spring Java/spring-webflux Java/t-io Java/tapestry Java/undertow Java/undertow-jersey Java/vertx Java/vertx-web"'
     - 'TESTDIR="Java/javalin Java/jawn Java/ratpack Java/wizzardo-http"'
     - 'TESTDIR="Java/play1 Java/play2-java Java/wildfly-ee"'
+    - 'TESTDIR="Java/simple-server"'
```

Comment:
You can add Java/simple-server to the end of this line:

```
- 'TESTDIR="Java/spring Java/spring-webflux Java/t-io Java/tapestry Java/undertow Java/undertow-jersey Java/vertx Java/vertx-web"'
```

They all build from the same base docker image and will utilize the docker cache on full travis runs